build.sbt

```diff
@@ -8,7 +8,7 @@ scalaVersion := "2.12.11"
  
 libraryDependencies ++= Seq(
   "com.mitre" % "dfdl-jpeg" % "0.0.1",
-  "org.apache.daffodil" %% "daffodil-tdml-processor" % "2.6.0" % "test",
+  "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.2.0-SNAPSHOT" % "test",
```

Comment:
Just curios, does this actually depend on on features in 3.2.0? We might want to only bump this to 3.1.0 so that non daffodil devs can easily test this.

Reply:
Right, this doesn't need 3.2.0-SNAPSHOT.  Will change to 3.1.0.